packages/i18n/data/core.json

```diff
@@ -53,7 +53,7 @@
   "FailedAuthentication.title": "We are unable to authorize you to use the Merchant Center.",
   "NavBar.MCSupport.title": "Support",
   "Notification.hideNotification": "Hide notification",
-  "PageNotFound.paragraph1": "The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again.\n\nPlease contact your system administrator or the commercetools <a>Help Desk</a> if you have any further questions.",
+  "PageNotFound.paragraph1": "The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again. Please contact your system administrator or the commercetools <a>Help Desk</a> if you have any further questions.",
```

Comment:
Hmm unrelated to this PR, but we should fix this. Let's revert this change please.

Reply:
done